### PR TITLE
Added an exclamation point (!)

### PR DESCRIPTION
**Pull request by LEQUI000**
I wanna be excited! added an exclamation point to the banner

### CODE REVIEW

**Comment by Schobbish** [comment]
great_find_1024x

**Comment by LEQUI000** [comment]
We can decide whether or not to add an exclamation or not.

**Comment by democat3457** [comment]
(for posterity) imo the period is more professional

File: _posts/data-announces-formation.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ author:
 ogImage:
   url: "/assets/blog/data-announces-formation/cover.jpg"
 banner: 
-  text: "Read our latest press release announcing our formation."
+  text: "Read our latest press release announcing our formation!"
```
Comment (Schobbish) [inline _posts/data-announces-formation.md]: Replace exclamation mark with period

**Review by Schobbish — CHANGES_REQUESTED** [review]

**Review by Schobbish — APPROVED** [review]

**Pull request merged**